provision/testdata/flavor_openshift_49_openstack_tar/cluster-network-06-PriorityClass-acicni-priority.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1beta1
```

Comment:
Since 4.9 uses k8s 1.22, the above needs to be fixed in the template (only for 4.9), please see:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/#priorityclass-v122

Reply:
make sense yes. Following it